config/routes.rb

```diff
@@ -2,5 +2,9 @@
   scope "(:locale)", locale: /en|vi/ do
     get "/login", to: "sessions#new"
     get "/signup", to: "users#new"
+    get "/about", to: "static_pages#about"
+    get "/contact", to: "static_pages#contact"
+  	get "/service", to: "static_pages#service"
```

Comment:
indent

Reply:
cái này e fix rồi mà push lên nó vẫn hiện vậy :(((

Reply:
@huuphuc93 I see that, it is not fixed. Please remove extra white space you added and align it with line 4

Reply:
@mdmahmudurrahman  i fixed that in my local, but when i push here, it don't work